### PR TITLE
[b/376629469] Upgrade avro library to version 1.11.4

### PR DESCRIPTION
**Pull request by kaxuna**
apache avro version set to 1.11.4 and lockfiles updated

### CODE REVIEW

File: dumper/lib-ext-hive-metastore/gradle.lockfile [hunk 1]
```diff
@@ -250,7 +249,7 @@ org.apache.twill:twill-discovery-core:0.6.0-incubating=hive312RuntimeClasspath
 org.apache.twill:twill-zookeeper:0.6.0-incubating=hive312RuntimeClasspath
 org.apache.yetus:audience-annotations:0.5.0=hive312RuntimeClasspath
 org.apache.zookeeper:zookeeper:3.5.3-beta=hive312RuntimeClasspath
-org.checkerframework:checker-qual:3.48.2=annotationProcessor,testAnnotationProcessor,testFixturesAnnotationProcessor
+org.checkerframework:checker-qual:3.33.0=annotationProcessor,testAnnotationProcessor,testFixturesAnnotationProcessor
```
Comment (vladislav-sidorovich) [inline dumper/lib-ext-hive-metastore/gradle.lockfile:252]: it's unexpected to see version decrease... Is it correct?

Comment (kpiotrowski) [reply dumper/lib-ext-hive-metastore/gradle.lockfile:252]: agree, worth to check why this version was downgraded

Comment (kaxuna) [reply dumper/lib-ext-hive-metastore/gradle.lockfile:252]: I can not produce anything other than that. Can you please pull the branch, delete lock files  build and produce lock files?

I do it with  gradle dependencies --write-locks

Comment (vladislav-sidorovich) [reply dumper/lib-ext-hive-metastore/gradle.lockfile:252]: Explicit version is defined in root gradle/libs.versions.toml:11

**Review by vladislav-sidorovich — APPROVED** [review]

**Pull request merged**